io.github.gssh_linux.gssh.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.gssh_linux.gssh
+runtime: org.gnome.Platform
+runtime-version: '44'
+runtime-options:
+sdk: org.gnome.Sdk
+command: runner.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --env=GI_TYPELIB_PATH=/app/lib/girepository-1.0
+modules:
+  - name: Gssh
+    buildsystem: simple
+    build-commands:
+      - install -D Flatpak/runner.sh /app/bin/runner.sh
+      - install -D main.py /app/bin/main.py
+      - install -D Flatpak/.desktop /app/share/applications/io.github.gssh_linux.gssh.desktop
+      - install -D Flatpak/appicon.png /app/share/icons/hicolor/128x128/apps/io.github.gssh_linux.gssh.appicon.png
+      - install -D Flatpak/io.github.gssh_linux.gssh.appdata.xml /app/share/metainfo/io.github.gssh_linux.gssh.appdata.xml
+
+    sources:
+      - type: git
+        url: https://github.com/gssh-linux/gssh.git
+        tag: 1.2.2
+  - name: vte
+    buildsystem: meson
+    config-opts:
+      - "-Dglade=false"
+      - "-Dgtk3=true"
+      - "-Dgtk4=false"
+      - "-Dvapi=false"
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/vte.git
+        tag: vte-0-74
+
+  - name: org.gnome.Platform
+    version: 44
+    runtime: org.gnome.Platform
+    sdk: org.gnome.Sdk
+    override: true
+    build-options:
+      # Add the Vte extension
+      - --share=lib/vte 
```

Comment:
not sure what you are doing here but this is invalid.

```
flatpak-builder:2): flatpak-builder-WARNING **: 00:49:15.365: Unknown property runtime-options for type BuilderManifest
(flatpak-builder:2): flatpak-builder-WARNING **: 00:49:15.365: Unknown property version for type BuilderModule
(flatpak-builder:2): flatpak-builder-WARNING **: 00:49:15.365: Unknown property runtime for type BuilderModule
(flatpak-builder:2): flatpak-builder-WARNING **: 00:49:15.365: Unknown property sdk for type BuilderModule
(flatpak-builder:2): flatpak-builder-WARNING **: 00:49:15.365: Unknown property override for type BuilderModule
(flatpak-builder:2): Json-WARNING **: 00:49:15.365: Failed to deserialize "build-options" property of type "BuilderOptions" for an object of type "BuilderModule"
```

Reply:
Useless so I removed it.